packages/react-storybook/Spec.md

```diff
@@ -0,0 +1,63 @@
+# @fluentui/react-storybook Spec
```

Comment:
Can you please delete this file? It's absolutely useless there 😜

Reply:
I know it is :D we need better generators ;)